src/ui/assets/AssetsListGridView.js

```diff
@@ -3,14 +3,13 @@ import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
 import { DropdownKebab, MenuItem } from 'patternfly-react';
 
-const AssetsListGridView = ({ assets }) => {
+const AssetsListGridView = ({ assets, domain }) => {
   const gridItems = assets.map((asset) => {
     const { versions } = asset;
     const fileType = versions == null ? 'file' : 'image';
-    // console.log(versions[3]);
     const previewRender = fileType === 'image' ? (
       <img
-        src={`http://localhost:8080${versions[3].path}`}
+        src={`http://${domain}${versions[3].path}`}
```

Comment:
make sure your `domain` var comes with the `http` protocol to avoid placing `http://` on `src` attribute. Some Entando instances are SSH protected so it's important to use the URL api I suggested to avoid hassle.